src/test/cpp/compare/out/winrt/Component.Fast.h

```diff
@@ -0,0 +1,304 @@
+﻿// WARNING: Please don't edit this file. It was generated by C++/WinRT v2.0.000000.0
+#pragma once
+#include "winrt/impl/Component.2.h"
+#include "winrt/impl/Component.Fast.2.h"
+#include "winrt/Component.h"
+namespace winrt::impl
+{
+    template <typename D> hstring consume_Component_Fast_IFastClass<D>::First() const
+    {
+        void* result;
```

Comment:
This is an example of the call site optimization that reduces the size of the shims by around 35% by eliminating the exception frame previously required when the result variable matched the return type.

Reply:
sweet